components/LinkButton/LinkButton.jsx

```diff
@@ -1,31 +1,22 @@
 import PropTypes from 'prop-types';
 import Router from 'next/router';
 
-const LinkButton = ({ label, route }) => (
-  <div>
+const LinkButton = ({ label, route }) => {
+  const handleLink = (url) => window.open(url, '_blank');
+  const isExternal = route && route.includes('google');
```

Comment:
I would check if it's a relative or absolute URL rather than checking if contains "google".

```suggestion
  const isExternal = route && route.includes('https://');
```